richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/model/RichTextState.kt

```diff
@@ -551,14 +551,28 @@ class RichTextState internal constructor(
     }
 
     fun toggleUnorderedList() {
-        val paragraph = getRichParagraphByTextIndex(selection.min - 1) ?: return
-        if (paragraph.type is UnorderedList) removeUnorderedList()
-        else addUnorderedList()
+        val firstParagraph = getRichParagraphByTextIndex(selection.min - 1) ?: return
```

Comment:
Why you are getting the first paragraph here, the paragraph list should be enough.
And if the list if empty, you can return

Reply:
As with Ordered List, changed